app/mailers/thing_mailer.rb

```diff
@@ -1,9 +1,15 @@
 class ThingMailer < ActionMailer::Base
   default from: 'adoptadrain@codeforsanfrancisco.org'
 
+  def adopted_confirmation(thing)
+    @thing = thing
+    @user = thing.user
+    mail(to: @user.email, subject: ['Thank you for Adopting a Drain in San Francico'])
+  end
+
   def reminder(thing)
     @thing = thing
     @user = thing.user
-    mail(to: thing.user.email, subject: ['Remember to shovel', thing.name].compact.join(' '))
+    mail(to: @user.email, subject: ['Remember to shovel', @thing.name].compact.join(' '))
```

Comment:
Should we change `shovel` to something else?

It might be nice to try to move some of this copy into the database to more easily contribute back changes to other `adopt-a-thing` projects eventually.